### DISCUSSION TITLE
删除英文摘要页

### DISCUSSION THREAD

**Discussion by AsunaOvO** [sjtug/SJTUThesis#1006] [提问] [answered]
请问应该如何删除英文摘要页？

**Comment by LogCreative** [comment]
如果你指的是英文大摘要的话，注释掉这一行

https://github.com/sjtug/SJTUThesis/blob/0eff2175eb573293a1d274262a2325c6ab5c41c0/main.tex#L90

**Reply by AsunaOvO** [reply]
你好，已经注释掉了，但是摘要还是有显示
![image](https://github.com/sjtug/SJTUThesis/assets/107908449/33fec5a9-725a-4847-bb38-544bd2473f4c)

**Reply by LogCreative** [reply, marked as answer]
如果你指的是前面的英文摘要的话，有没有特殊的理由需要这么做呢，如果是标准的学位论文这部分是需要的。

**Reply by AsunaOvO** [reply]
我们是用于课程论文，请问有什么方法删掉吗

**Reply by LogCreative** [reply]
注释掉这一段即可

https://github.com/sjtug/SJTUThesis/blob/0eff2175eb573293a1d274262a2325c6ab5c41c0/contents/abstract.tex#L14-L24

**Reply by AsunaOvO** [reply]
好的，感谢: - )
